src/templates/navigation/footer.pug

```diff
@@ -18,7 +18,7 @@ footer#footer-main.position-relative
       hr.divider.divider-fade.divider-dark.my-5
       .row
         .col-lg-4.mb-5.mb-lg-0
-          a(href='index.html')
+          a(href='https://zerobase.io/')
```

Comment:
Lets change this to `a(href='/') ` since staging will need to point back to staging